agr_api/src/main/java/org/alliancegenome/api/controller/GeneController.java

```diff
@@ -185,7 +187,7 @@ public Response getAllelesPerGeneDownload(String id,
         pagination.addFieldFilter(FieldFilter.DISEASE, disease);
         pagination.addFieldFilter(FieldFilter.VARIANT_TYPE, variantType);
         pagination.addFieldFilter(FieldFilter.PHENOTYPE, phenotype);
-        pagination.addFieldFilter(FieldFilter.VARIANT_CONSEQUENCE, consequence);
+        pagination.addFieldFilter(FieldFilter.MOLECULAR_CONSEQUENCE, consequence);
```

Comment:
I would guess that `consequence` should be `molecularConsequence`?

Reply:
Yes, let's make it consistent.